bs-config.js

```diff
@@ -0,0 +1,21 @@
+/*
+ | Browser-sync config file
+ | For up-to-date information about the options:
+ |   http://www.browsersync.io/docs/options/
+ | */
+ module.exports = {
+  "ui": false,
+  "files": ['dist/**', 'example/**', 'docs/**', 'themes/**'],
```

Comment:
I'm planning on moving the examples/ folder to introjs-website repo. I think we should keep this repo for Intro.js codebase and the `introjs-website` for docs, examples, etc.

Reply:
Ah, I was wondering about that.  That makes this whole PR dead.  I can make new instructions once those changes are made I guess.  We could run rollup -c -w in this repo, and then browsersync in the website, with yarn link or something?  Cool to close this PR?

Reply:
Actually what happens with the netlify deploys if the example dir is moved?

Reply:
That's a good question. I did not consider the Netilfy deploys :thinking:

Reply:
git sub-modules? :fearful:

Reply:
Not sure if there are any good solutions.  Symlinking and string replacements (for URL's) come to my mind.  I guess it's a good question as to what the examples pages are for.  Are they for the website? or for development?

Reply:
both? I thing the end goal is to have the examples folder on the website but it's also nice to have that folder in the main repo (helps with Netlify deployments as you mentioned).